cfgov/unprocessed/css/print.less

```diff
@@ -28,25 +28,37 @@
     - cfgov-print
 */
 
+.respond-to-print({
+    a::after {
+        word-break: break-all;
+    }
 
-/* topdoc
-  name: Print header
-  notes:
-    - "Separates print header stuff from the rest of the page content."
-    - "Used on /about-us/the-bureau/leadership-calendar/print/"
-  family: cfgov-print
-  patterns:
-    - name: .print-header
-      markup: |
-        <div class="print-header">
-            There should be a line underneath me =D
-        </div>
-  tags:
-    - cfgov-print
-*/
+    a[href*="://"]::after {
+        // Output href in parentheses for links with a protocol already there
+        content: " (" attr(href) ")";
+    }
+
+    a[href^="/"]::after {
+        // Output root-relative hrefs in parentheses with our domain prepended
+        content: " (https://www.consumerfinance.gov" attr(href) ")";
+    }
+
+    a[href^="#"] {
+        // Style in-page anchor links (like page TOCs) to not look like links
+        color: @dark-gray;
+        border-bottom-width: 0;
+    }
+
+    .o-video-player_play-btn {
+        display: none !important;
```

Comment:
It would be helpful to add a code comment above any `!important` to note why it's required.

Reply:
:+1: Done in f63b2f9351845a02dda5c6db5ee609c02be8a1db